content/yaml-publishing/distribution.md

```diff
@@ -127,11 +131,15 @@ publishing:
 
 ```
 
-#### Post-processing of App Store Connect distribution
+{{<notebox>}}
+Multiple Apple Developer Portal accounts can be connected for publishing other iOS apps with different workflows
```

Comment:
To use different Apple Developer Portal accounts for publishing your iOS apps, set up separate workflows.